### PR TITLE
Implements the showOnHover feature described in author's documentation.

### PR DESCRIPTION
**Pull request by WilliamRADFunk**
Changes to code involved:

1. an upgrade to the addInitHook calls for both polyline and circle in main file..

2. added examples on the example page to display the hover ability and implementation in both polyline and circle.

3. added unit tests to cover showOnHover feature in both polyline and circle.

### CODE REVIEW

**Comment by WilliamRADFunk** [comment]
@perliedman Let me know if this meets your criteria.

File: package.json [hunk 1]
```diff
@@ -15,6 +15,9 @@
     "path"
   ],
   "author": "Per Liedman <per@liedman.net>",
+  "contributers": [
```
Comment (perliedman) [inline package.json]: Should be:

```suggestion
  "contributors": [
```

...right? (https://docs.npmjs.com/files/package.json#people-fields-author-contributors)

**Comment by WilliamRADFunk** [comment]
@perliedman : Nice catch on the spelling. I committed your suggestion, so should be good to go as you say.

**Comment by perliedman** [comment]
Nice, thanks again! I'll make a new release.

**Comment by perliedman** [comment]
Released as 1.4.0!

**Pull request merged**